src/cr/cube/crunch_cube.py

```diff
@@ -453,8 +531,24 @@ def _mr_margin(self, axis, weighted, adjusted):
 
         return np.sum(margin, 1 - self.mr_dim_ind) if axis is None else margin
 
+    def _inflate_dim(self, array, axis):
+        # Explicitly check if axis is tuple (which could be the case for doing
+        # cell percentages across 3D cube, when the axis is set to (1, 2)).
+        # Python 3 doesn't support tuple to int conversion, and we have to
+        # check it manually.
+        if (axis and not isinstance(axis, tuple) and
+                axis > 0 and len(array.shape) == 1):
+            # If any of the dimensions has only one element, it's flattened
+            # from the resulting array (as a part of the MR pre-processing).
+            # This can lead to a potential inconsistency between dimensions
+            # and axes, and we need to restore one dimension in this case.
+            array = array[:, np.newaxis]
+
+        return array
+
     def _margin(self, axis=None, weighted=True, adjusted=False,
                 include_transforms_for_dims=None, prune=False):
+        '''Calculate cube margin.'''
```

Comment:
I wouldn't use docstring for internal functions... also this is pretty self explanatory from the name of the function.